tutorials/quickstart.eve.md

```diff
@@ -0,0 +1,90 @@
+# Quickstart
+
+In this document we're going to give a 5 minute introduction to concepts essential to Eve. If you've never used Eve before, you're in the right place.
+
+## Adding records to Eve
+
+Eve represents data as records, key value pairs attached to a unique ID. Eve programs are made up of small composable blocks that search for and create records. Let's start with a block that adds a record to Eve:
+
+```eve
+commit
+  [#greeting text: "hello world"]
+```
+
+This record is tagged `#greeting`, and has an attribute "text" with the value "hello world". Although tags are attributes like any other,they have special syntax because they are used often.
+
+## Finding records in Eve
+
+We can search for records, and then bind new records to those that are found. Eve finds every record matching the supplied patterns, and then binds records dependent on them. If no records match the search, then the block does not run at all. A block will only run if every record in a search matches at least one result. Let's search for the `#greeting` we just committed, and then display it in a div:
+
+```eve
+search
+  [#greeting text]
+
+bind
+  [#ui/div text]
+```
+
+Names are identities in Eve; because they have the same name, the `text` in `#greeting` is bound to the `text` in the div. Go ahead and add another `#greeting` record to the first block to see what happens when more than one record matches the search. 
```

Comment:
I don't think "names are identities in Eve" is a good way to express this. I'm struggling to find something equally concise though. I also struggled to express this in the language reference. I eventually just described it mechanically since I *think* the context is sufficient so long as I don't cloud it with words that are confusing.